tools/softirqs.py

```diff
@@ -22,6 +23,7 @@
 # arguments
 examples = """examples:
     ./softirqs            # sum soft irq event time
+    ./softirqs -C         # show soft irq event counts
```

Comment:
to avoid referring to count, just say 'show the number of soft irq events'. The '-C' option is okay to me.

Reply:
Done. btw, args.count is conflicted with positional argument 'count', use args.events instead.